plugins/eos-updater/gs-plugin-eos-updater.c

```diff
@@ -767,6 +769,50 @@ gs_plugin_eos_updater_dispose (GObject *object)
 	G_OBJECT_CLASS (gs_plugin_eos_updater_parent_class)->dispose (object);
 }
 
+/* Submit a metrics event about whether parental controls are enabled,
+ * which gives us an overview of how many users are using parental
+ * controls.
+ *
+ * See: https://phabricator.endlessm.com/T28741 */
+static void
+report_parental_controls_metric (GCancellable *cancellable)
+{
+	g_autoptr(GDBusConnection) system_bus = NULL;
+	g_autoptr(MctManager) manager = NULL;
+	g_autoptr(MctAppFilter) filter = NULL;
+	g_autoptr(GError) metrics_error = NULL;
+	gboolean parental_controls_enabled = FALSE;
+
+#define GS_PARENTAL_CONTROLS_USAGE_EVENT "c227a817-808c-4fcb-b797-21002d17b69a"
```

Comment:
I'd love to move this somewhere else that runs on a timer. But we can carry this forward for now.